### DISCUSSION TITLE
Welcome to MetaResearch Journal Club!

### DISCUSSION THREAD

**Discussion by KarolaKirsanow** [protocol/research#36] [MetaResearch Journal Club]
📚 The PL Journal Club will focus on **MetaResearch**: the study of research itself - scientific methods, evaluation techniques, incentives, and more. We'll be reading and discussing ways of accelerating our research and research engineering, and learning about the successful R&D initiatives of the past and present. Tools, mental models, evaluation frameworks - the works!

Please use this board to post and discuss suggested readings!
We meet here: https://protocol.zoom.us/j/91732611849 

👉 You can sign up to select a topic and lead a discussion [here](https://docs.google.com/spreadsheets/d/1A-rwJ2L0D7ujKiaKa8QqrKIUSMB5vdVLcKtVyOAguwU/edit?usp=sharing)

**Comment by KarolaKirsanow** [comment]
Notes from an earlier discussion on the Journals and Conferences of the Future:

[Meta-Journal_Club_the_knowledge_dissemination_models_of_the_future.pdf](https://github.com/protocol/research/files/6514409/Meta-Journal_Club_the_knowledge_dissemination_models_of_the_future.pdf)

**Reply by redransil** [reply]
Could we collaboratively put together a lit review of new publishing and citation tools/methodologies using the projects mentioned in these notes as a starting point? I'm especially interested in how traditional academic tools interface with (1) newer systems for publishing and citing research code like the DOI system, Zenodo, [Code Ocean](https://codeocean.com/), the [Wolfram Function Repo](https://resources.wolframcloud.com/FunctionRepository/), (2) other granular publication methods like micropublications and [Research Objects](https://www.researchobject.org/), (3) digital lab notebooks and (4) lab automation platforms like [Emerald Cloud Lab](https://www.emeraldcloudlab.com/) and [Strateos](https://strateos.com/)

**Reply by KarolaKirsanow** [reply]
I think there was enough async interest in this topic to revisit it a second time.

**Reply by joelchan** [reply]
@redransil I'd be interested in something like this!

**Reply by metasj** [reply]
+ in development: [Wikifunctions](https://meta.wikimedia.org/wiki/Abstract_Wikipedia/Function_model) / Wikilambda, and a U.Mich function repository

**Reply by metasj** [reply]
Other sources to add to this list?  [HuggingFace models](https://huggingface.co/models), [Problem-generator-functions](https://www.wolframalpha.com/problem-generator/), Pipes ([IFTTT](https://ifttt.com/explore), Import.io, ...) ?  

^: Perhaps each session we could have a [lit review] section, w/ lightning-talks about areas that need fleshing out, as well as a [discussion topic] section w/ a more focused topic.

**Reply by KarolaKirsanow** [reply]
Do you want to start us off with a few pieces to focus on  and a lightning talk ⚡ ?

**Reply by metasj** [reply]
I feel I don't have a sufficiently wide view of the space yet!  But could talk about Wikifunctions and Pipes...

**Reply by KarolaKirsanow** [reply]
Would this [working paper](https://arxiv.org/pdf/2004.04733.pdf) be a relevant starting point for the discussion?

**Comment by redransil** [comment]
As far as suggested reading, @davidad suggested Poston's [Reason and Explanation - A Defense of Explanatory Coherentism](https://www.palgrave.com/gp/book/9781137012258#) a few months ago. I started it and am enjoying it - would be great to discuss!

**Comment by KarolaKirsanow** [comment]
We could do a session on the relationship of UX/UI design to the [future of scientific publication](https://royalsociety.org/blog/2015/05/fssc-interface-design-and-collective-intelligence/) and [explanation](https://michaelnielsen.org/reinventing_explanation/index.html). 

Andy Matuschak and Michael Nielsen's 2019 [essay](https://numinous.productions/ttft/) would be a useful starting point for a discussion of tooling-for understanding-and-dissemination.

Cindy asked a great question in our session on Journals of the Future: _how do you optimize for fun?_

Related: 

- Alan Kay on [The Future of Reading Depends on the Future of Learning Difficult to Learn Things](http://www.vpri.org/pdf/future_of_reading.pdf) and the [Dynabook](https://dl.acm.org/doi/pdf/10.1145/61975.66919)

**Comment by joelchan** [comment]
Suggested readings (in no particular order):

- [Bret Victor - Magic Ink: Information Software and the Graphical Interface](http://worrydream.com/MagicInk/#p134): mind-expanding ideas around the medium itself for complex information  
- [Max Krieger - Chatting with Glue: Cognitive Tools for Augmented Conversation](http://a9.io/glue-comic/): mind-expanding ideas around the medium itself for discussion/conversation  
- [Allen Renear & Carole Palmer (2009) - Strategic Reading, Ontologies, and the Future of Scientific Publishing ](https://science.sciencemag.org/content/325/5942/828): bold predictions of semantic reading by 2019 (published in Science!), which... haven't yet come to pass, but the history and vision is worth a read. Last author is major player in library information science (LIS) literature on scholarly information behavior (scholarly workflows, primitives) and interdisciplinarity.  
- [Anna de Liddo, Agnes Sandor, and Simon Buckingham Shum (2012) - Contested Collective Intelligence: Rationale, Technologies, and a Human-Machine Annotation Study](http://oro.open.ac.uk/31052/): good entry point into another stream of work on discourse-centric models of scholarly collective sensemaking and publishing, with a special helpful twist emphasizing the *contested* nature of discourse (so a hard pushback against centralized "single source of truth" ontologies etc.). the last author (Simon Buckingham-Shum) is/was a major player in alternative models of scholarly communication and collective intelligence and deeply rooted in the HCI/CSCW literature on how hard this problem is (now mostly working in the learning analytics space, hopefully not because the problem is intractable!!)  
- [Anita de Waard (2010) - From Proteins to Fairytales: Directions in Semantic Publishing](https://www.semanticscholar.org/paper/From-Proteins-to-Fairytales%3A-Directions-in-Semantic-Waard/32b275e519d821de9037915360fea82ee6c2b5cf): similar to Renear, but with more details, focused on biomed, and makes a nice case for  discourse-centric representations (not just RDF triples)
- [Dave Randall et al (2011) - Distributed ontology building as practical work](https://linkinghub.elsevier.com/retrieve/pii/S1071581911000024): deep dive into the use of a major tool (Protege) for building ontologies.   
- [Susan Halford et al (2013) Digital Futures? Sociological Challenges and Opportunities in the Emergent Semantic Web](https://doi.org/10.1177/0038038512453798): Expands on Randall's themes of the fraught nature of semantic publishing in much more depth and breadth

**Reply by KarolaKirsanow** [reply]
These look great!

**Comment by KarolaKirsanow** [comment]
What we discussed on Monday:

# How Can We Develop Transformative Tools for Thought?

## *the most powerful tools for thought express deep insights into the underlying subject matter*

![Untitled](https://user-images.githubusercontent.com/25029171/121251806-8fa54a00-c8a7-11eb-942a-1f3ef178f3c4.png)


Hildegard von Bingen, 12thc, *Columba aspexit* - neumes notation

![Untitled 1](https://user-images.githubusercontent.com/25029171/121251837-992eb200-c8a7-11eb-89ad-7f641c85832c.png)

Ludwig van Beethoven 19thc, *9th Symphony* 

## many tools for thought are public goods



![Untitled 2](https://user-images.githubusercontent.com/25029171/121251886-a51a7400-c8a7-11eb-82f2-e923caaa2a58.png)
![Untitled 3](https://user-images.githubusercontent.com/25029171/121251891-a77cce00-c8a7-11eb-93b3-ddf3f8935cca.png)


## Why *do* books work?


![Untitled 4](https://user-images.githubusercontent.com/25029171/121251922-b1063600-c8a7-11eb-9969-3d8d45eddb8d.png)


- Memory is far more important than people tend to think
- Take emotion seriously

## The executable book

https://observablehq.com/embed/@bronna/parametric-seashell?cell=*

## *good tools for thought arise mostly as a byproduct of doing original work on serious problems*

![Untitled 5](https://user-images.githubusercontent.com/25029171/121251947-b794ad80-c8a7-11eb-8f90-30e83f204702.png)
Necessary...

![Untitled 6](https://user-images.githubusercontent.com/25029171/121251958-bcf1f800-c8a7-11eb-9afd-0a9ce80fcd28.png)
Python_scripts

...but not sufficient

**Reply by KarolaKirsanow** [reply]
Further notes on the discussion:
- what are the problems we want to solve with the tools we want to build?
- how does one go about developing a "[research taste](https://colah.github.io/about.html)"
- what is the role of fun? lessons from music and sport?
- when can we consider two similar bits of evidence equivalent?
- can claims be equivalent if they originated from or relate to different contexts?

**Comment by KarolaKirsanow** [comment]
Suggestions for 12 July:
- [McMahan, P., & Evans, J. (2018). Ambiguity and engagement. American Journal of Sociology, 124(3), 860-912](https://petermcmahan.com/pdf/McMahan_Evans_2018_Ambiguity%20and%20Engagement.pdf) ✅ 
- [Meadows, D. H. (1999). Leverage points: Places to intervene in a system.](https://donellameadows.org/archives/leverage-points-places-to-intervene-in-a-system/)

**Comment by KarolaKirsanow** [comment]
Possibilities for 2 August:

-  Swanson, D. - [Undiscovered Public Knowledge](http://abel.lis.illinois.edu/tutorial/swanson_libquart_1986.pdf) and [Rediscovering Don Swanson: the Past, Present and Future of Literature-Based Discovery](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5771422/)
- Franzoni, Stephan, & Veugelers -  [Funding Risky Research](https://www.nber.org/papers/w28905)
- Franzoni & Stephan - [Uncertainty and Risk-Taking in Science: Meaning, Measurement and Management](https://www.nber.org/papers/w28562)

**Reply by metasj** [reply]
+ the [Overedge Catalog](https://arbesman.net/overedge/) (+ modes of risk-taking in each)

**Reply by cindywu** [reply]
+1 for Franzoni, Stephan, & Veugelers or Franzoni & Stephan

Having @maxkreiger present his own work and lead discussion was really fun. If the group wanted to do something similar, I could ask Chiara Franzoni if she wants to present and lead discussion on one of the papers above.

**Comment by cindywu** [comment]
Suggested reading: 

- Any chapter from [Gaming the Metrics: Misconduct and Manipulation in Academic Research](https://direct.mit.edu/books/book/4598/Gaming-the-MetricsMisconduct-and-Manipulation-in)

**Comment by KarolaKirsanow** [comment]
**Cycles of Invention and Discovery**
Questions from 4 October:
- is it fair to credit Vannevar Bush with creating the applied/basic science dichotomy? 
- p.60: "the innovation culture of IBM emphasized producing results, even if they might not be commercially useful to the company" -- how important is the "slack" provided  by monopoly position or rents (AT&T) or huge profit margins (IBM) in enabling this type of  long-term vision? It sounds like AT&T considered itself as part of the basic national infrastructure

**Comment by KarolaKirsanow** [comment]
[Science, the Endless Frontier](https://www.nsf.gov/od/lpa/nsf50/vbush1945.htm) by Vannevar Bush

![image](https://user-images.githubusercontent.com/25029171/135901586-8d692ad5-4128-4678-be90-76721d7e5d89.png)

**Reply by KarolaKirsanow** [reply]
Surprised to see that there are no public hypothes.is annotations on this piece

**Comment by KarolaKirsanow** [comment]
[Mirror-Image  Twins:  The Communities of 
Science  and Technology  in 19th-Century 
America](https://www.rodrigofernos.org/scirev/lecturas/12_layton_mirror_image.pdf) by Edwin Layton

**Comment by KarolaKirsanow** [comment]
[Galileo’s stream: A framework for understanding
knowledge production](http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.720.8692&rep=rep1&type=pdf) by J.Y. Tsao et al.

**Comment by KarolaKirsanow** [comment]
Also of interest: Carlotta Perez's work on [Technological Revolutions and Financial Capital](https://stratechery.com/2021/the-death-and-birth-of-technological-revolutions/)
